tests/js/snapshots.test.js

```diff
@@ -7,3 +7,9 @@ test.describe("footer", () => {
     await expect(footer).toHaveScreenshot();
   });
 });
+
+test("tutorials do not have purple border", async ({ page }) => {
+  await page.goto("sphinx_guide/notebook_gallery.html");
+  const tutorial = page.locator("div.nbsphinx-gallery");
```

Comment:
As seen in the screenshot, this captures the entire nbsphinx gallery. More precise would be just a single tutorial itself with `div.nbsphinx-gallery a`.

Usually I think it's better to have more precise tests, but here it's maybe useful to make sure we're aware of if we ever accidentally change tutorials like spacing between the elements? Maybe I should add a second tutorial to the gallery?